yaml/shared/src/main/scala/org/virtuslab/yaml/internal/load/reader/ReaderCtx.scala

```diff
@@ -18,6 +18,7 @@ case class ReaderCtx(reader: Reader) {
 
   def indent: Int                     = indentations.lastOption.getOrElse(-1)
   def addIndent(newIndent: Int): Unit = indentations.append(newIndent)
+  def removeLastIndent(): Unit        = indentations.removeLast()
 
   def checkIndents(current: Int): Unit =
```

Comment:
`checkIndents` could return `List[Token]` instead of appending those token itself